src/platform/Kconfig

```diff
@@ -304,6 +304,13 @@ config MAX_CORE_COUNT
 	help
 	  Maximum number of cores per configuration
 
+config MP_NUM_CPUS
```

Comment:
Is there any way we can avoid the duplication with Zephy ? i.e.  could the Kconfig item depend on XTOS == true and would reusing the Zephyr Kconfig symbol for xtos help here ? Just want to avoid any confusion around Zephyr Kconfig (which we had in the past and had to fix). @kv2019i any comments ?